x-pack/plugins/reporting/server/config/schema.ts

```diff
@@ -33,9 +33,13 @@ const KibanaServerSchema = schema.object({
 const QueueSchema = schema.object({
   indexInterval: schema.string({ defaultValue: 'week' }),
   pollEnabled: schema.boolean({ defaultValue: true }),
-  pollInterval: schema.number({ defaultValue: 3000 }),
+  pollInterval: schema.oneOf([schema.number(), schema.duration()], {
+    defaultValue: moment.duration('3s'),
+  }),
   pollIntervalErrorMultiplier: schema.number({ defaultValue: 10 }),
-  timeout: schema.number({ defaultValue: moment.duration(2, 'm').asMilliseconds() }),
+  timeout: schema.oneOf([schema.number(), schema.duration()], {
+    defaultValue: moment.duration('2m'),
```

Comment:
@joelgriffith I was able to use object values that make more sense as the defaults. I just had not been passing the right kind of object on my first attempt.

Thanks for the tip!

Reply:
Sweet!